config/karma.conf.js

```diff
@@ -94,25 +98,39 @@ module.exports = function (config) {
           }
         },
     */
-    browsers: ['PhantomJS_custom'],
+    browsers: ['ChromeNoSandboxHeadless'],
     customLaunchers: {
-      'PhantomJS_custom': {
-        base: 'PhantomJS',
-        options: {
-          windowName: 'alm-window',
-          settings: {
-            webSecurityEnabled: false
-          },
-        },
-        flags: ['--load-images=true'],
-        debug: true
+      ChromeNoSandboxHeadless: {
+        base: 'ChromeCanary',
+        flags: [
+          '--no-sandbox',
+          // See https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md
+          '--headless',
+          '--disable-gpu',
+          // Without a remote debugging port, Google Chrome exits immediately.
+          ' --remote-debugging-port=9222'
+        ]
       }
     },
-    phantomjsLauncher: {
-      // Have phantomjs exit if a ResourceError is encountered
-      // (useful if karma exits without killing phantom)
-      exitOnResourceError: true
-    },
+    // browsers: ['PhantomJS_custom'],
```

Comment:
Not sure who we do it in this project, but if that's dead code maybe we can simply remove it?